indico/htdocs/sass/_base.scss

```diff
@@ -15,10 +15,11 @@
  * along with Indico; if not, see <http://www.gnu.org/licenses/>.
  */
 
-@import 'base/palette';
-@import 'base/defaults';
-@import 'base/layout';
-@import 'base/reset';
-@import 'base/theme';
-@import 'base/typography';
-@import 'base/utilities';
+@import "base/palette";
```

Comment:
That doesn't make any sense. You should have broken dozens of pages using module-specific CSS that are simply importing `base`. Without this, how the in the world are they importing `palette` and the rest?